extensions/standard-processors/processors/ListenSyslog.cpp

```diff
@@ -204,7 +204,7 @@ void ListenSyslog::runThread() {
           logger_->log_debug("ListenSysLog client socket %d close", clientSocket);
           it = _clientSockets.erase(it);
         } else {
-          if ((uint64_t) (recvlen + getEventQueueByteSize()) <= _recvBufSize) {
+          if ((uint64_t) (recvlen + getEventQueueByteSize()) <= gsl::narrow<uint64_t>(_recvBufSize)) {
```

Comment:
same as above

Reply:
Updated in 267cda83a77a5a6d05f9df714d0c923814351826